Source/Core/Common/Network.cpp

```diff
@@ -27,7 +28,7 @@ void GenerateMacAddress(const MACConsumer type, u8* mac)
 		break;
 	}
 
-	srand((unsigned int)time(nullptr));
+	srand(Common::Timer::GetTimeMs());
```

Comment:
It would be nice to replace srand with the corresponding functionality in the `<random>` header.